### PR TITLE
fix(TBD-10689): deprecate old distributions

### PR DESCRIPTION
**Pull request by AlixMetivier**
**Please check if the PR fulfills these requirements**

- [ ] The commit(s) message(s) follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md#commit-message-format) ?
- [ ] Unit tests for the Java changes have been added (for bug fixes / features) ?
- [ ] TUJ for the JavaJet changes have been added (for bug fixes / features) ?
- [ ] Docs have been added / updated (for bug fixes / features) ?
- [ ] The new code does not introduce new technical issues

**What is the current behavior?** (You can also link to an open issue here)

**What is the new behavior?**

**BREAKING CHANGE**

If this PR contains a breaking change, please describe the impact and migration
path for existing applications.
If not please remove this section.

**Other information**:

### CODE REVIEW

**Review by firefryy — APPROVED** [review]
lgtm

**Review by cbadillo1603 — APPROVED** [review]
reviewed

File: main/plugins/org.talend.designer.components.bigdata/components/tSqoopExport/tSqoopExport_messages.properties [hunk 1]
```diff
@@ -140,26 +140,26 @@ DISTRIBUTION.ITEM.CUSTOM=Custom - Unsupported
...
+DB_VERSION.ITEM.HDP_1_2=Hortonworks Data Platform V1.2.0(Bimota) - Deprecated
+DB_VERSION.ITEM.HDP_1_3=Hortonworks Data Platform V1.3.0(Condor) - Deprecated
+DB_VERSION.ITEM.HDP_2_0=Hortonworks Data Platform V2.0.0(BigWheel) - Deprecated
+DB_VERSION.ITEM.HDP_2_1=Hortonworks Data Platform V2.1.0(Baikal) - Deprecated
+DB_VERSION.ITEM.HDP_2_2=Hortonworks Data Platform V2.2.0 - Deprecated
+DB_VERSION.ITEM.HDP_2_3=Hortonworks Data Platform V2.3.2 - Deprecated
+DB_VERSION.ITEM.Cloudera_CDH4=Cloudera CDH4.X (MR 1 mode) - Deprecated
+DB_VERSION.ITEM.Cloudera_CDH4_YARN=Cloudera CDH4.3+(YARN mode) - Deprecated
+DB_VERSION.ITEM.Cloudera_CDH5=Cloudera CDH5.0(YARN mode) - Deprecated
 DB_VERSION.ITEM.Cloudera_CDH5_1_MR1=Cloudera CDH5.1(MR 1 mode)
```
Comment (sponomarova) [inline main/plugins/org.talend.designer.components.bigdata/components/tSqoopExport/tSqoopExport_messages.properties:160]: What about this one? Still not deprecated?

**Review by cbadillo1603 — DISMISSED** [review]
reviewed

**Review by hliashchuk — APPROVED** [review]

**Review by hliashchuk — DISMISSED** [review]

**Comment by cbadillo1603** [comment]
I wonder if tHiveConnection should include the message as well.

**Review by hliashchuk — APPROVED** [review]

**Review by cbadillo1603 — DISMISSED** [review]
reviewed

**Review by cbadillo1603 — APPROVED** [review]
reviewed

**Review by hliashchuk — APPROVED** [review]

**Pull request merged**